### PR TITLE
(PC-4727) : android deploy in testing

### PR DESCRIPTION
**Pull request by DanielBaudry**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-4727

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Explained my technical strategy below if feature is complex.

If native code (ios/android) was modified, after the PR is merged go to master and upgrade the __app version__ (+1 patch):

(later there will be a script to embed these commands) 
- use `yarn version --patch` (it will create a commit and a tag)
- then `git push && git push origin <tag_name>`
- then use the deploy script (temporary -> waiting for CI/CD to be ready)

## Technical strategy

> _Insert technical strategy_

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -34,8 +34,53 @@ jobs:
...
+            - node_modules
+      - run:
+          name: Install gem dependencies
+          command: |
+            bundle install
+      - run:
+          name: Setup env variables
+          command: |
+            mkdir android/keystores
+            echo $ANDROID_KEYSTORE | base64 -d > android/keystores/testing.keystore
```
Comment (Giregm) [inline .circleci/config.yml]: très cool l'idée d'avoir le keystore en base 64 !

File: .circleci/config.yml [hunk 2]
```diff
@@ -34,8 +34,53 @@ jobs:
...
   version: 2
   commit:
     jobs:
-      - run-tests
+      #- run-tests
+      - deploy-android-testing:
+          filters:
+            branches:
+              only:
+                - pc-4727-android-deploy-in-testing
```
Comment (Giregm) [inline .circleci/config.yml]: Je ne comprends pas bien ce passage

On filtre sur cette branche ? (ou c'était juste un moyen de tester ?)

Comment (Giregm) [reply .circleci/config.yml]: Nevermind tu as changé ça :D

**Review by Giregm — APPROVED** [review]

**Review by BenStirrup — APPROVED** [review]
Ok pour moi ! 
On merge et on teste sur master ? :grin:

**Pull request merged**